MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -1189,8 +1189,9 @@ public string GetInputArgument(EncodingJobInfo state, EncodingOptions options, s
             {
                 var tmpConcatPath = Path.Join(_configurationManager.GetTranscodePath(), state.MediaSource.Id + ".concat");
                 _mediaEncoder.GenerateConcatConfig(state.MediaSource, tmpConcatPath);
-                arg.Append(" -f concat -safe 0 -i ")
-                    .Append(tmpConcatPath);
+                arg.Append(" -f concat -safe 0 -i \"")
+                    .Append(tmpConcatPath)
+                    .Append('"');
```

Comment:
I forgot to leave an extra whitespace
```suggestion
                    .Append("\" ");
```